packages/data-point/src/reducer-helpers/constant/constant.test.js

```diff
@@ -1,4 +1,31 @@
-const { ReducerConstant } = require("./constant");
+jest.mock("lodash/cloneDeep");
+const mockCloneDeep = require("lodash/cloneDeep");
+
+mockCloneDeep.mockImplementation(value => {
+  return jest.requireActual("lodash/cloneDeep")(value);
+});
```

Comment:
can you explain this code? not sure I understand - it creates a mock around the actual function?

Reply:
thi sis to use the original function duh.... . this could be simpler...